mozilla-release/browser/components/migration/MigrationUtils.jsm

```diff
@@ -1063,6 +1063,11 @@ var MigrationUtils = Object.freeze({
     // be valid for the history store.
     for (let pageInfo of pageInfos) {
       for (let visit of pageInfo.visits) {
+        // CLIQZ-SPECIAL: hack to transform date in required format when importing from FF
+        if(pageInfo.isFPMigrator) {
+          let { date: timeStamp } = visit;
+          visit.date = new Date(Math.floor(timeStamp/1000));
```

Comment:
@tsl143 you could do this in https://github.com/cliqz-oss/browser-f/pull/1231/files#diff-5b7f9e6a5e6d3a11ee370cac0bcc0d06R347 right? to avoid changing this common MigrationUtils.jsm